main.go

```diff
@@ -287,7 +301,28 @@ func confirmByUser(tty *tty.TTY) (bool, error) {
 	}
 }
 
-func createJob(f *os.File, job *batchv1.Job) error {
+type JobEditor interface {
+	EditJob(job *batchv1.Job) error
+}
+type InteractiveJobEditor struct {
+	Filename string
+}
```

Comment:
In golang, capital letter is significant. So, if you doesn't intend to make it public, you should start lowercase letter.

```suggestion
type jobEditor interface {
	EditJob(job *batchv1.Job) error
}
type interactiveJobEditor struct {
	filename string
}
```

Reply:
Thank you for your feedback! I appreciate the learning